bin/vgac2pps.py

```diff
@@ -45,7 +46,11 @@
     parser.add_argument('-all_ch', '--all_channels', action='store_true',
                         help="Save all 21 channels to level1c4pps file.")
     parser.add_argument('-n19', '--as_noaa19', action='store_true',
-                        help="Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. And apply SBAFs to the channels.")
+                        help=("Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. "
+                              "And apply SBAFs to the channels."))
+    parser.add_argument('--SBAF',
+                        help='SBAF version "v2" or "v3".',
```

Comment:
and I think it would better to remove one of '--as_noaa19' and '--SBAF', because now the user has the option to specify SBAF without specifying the '--as_noaa19' flag, which is kind of odd.
I think it would make sense to keep '--as_noaa19'  and there you have to set the SBAF choices.